Doc/library/importlib.rst

```diff
@@ -985,6 +985,17 @@ find and load modules.
       :exc:`ImportError` is raised.
 
 
+.. class:: NamespacePath(name, path, path_finder)
+
+   Represents a namespace package's path.
+
+   It uses the module *name* to find its parent module, and from there it looks
+   up the parent's :attr:`module.__path__`.  When this changes, the module's own
+   path is recomputed, using *path_finder*. The initial value is set to *path*.
```

Comment:
Can we perhaps mention the type of the path finder? (namely using some class Sphinx directive).

Btw, when you say that the initial value is set to *path* it seems that you are talking about the initial value of the path finder.

In addition, maybe mention that invalidating the caches forces the path to be recomputed (it might not necessarily be the case that the module parent's path has changed).

Finally, could you document the public methods of this new class? For instance there is no insert() method nor remove() as for lists. 

I think an example of how to use it could be useful though I don't know whether we want to burden the docs even more.